core/google/cloud/client.py

```diff
@@ -14,6 +14,8 @@
 
 """Base classes for client used to interact with Google Cloud APIs."""
 
+from pickle import PicklingError
```

Comment:
I explicitly went with a single-member import rather than a more general `import pickle` because I wanted it to be immediately clear when this is read months from now that the _only_ thing pickle is being used for is the exception class.